sys/androidmedia/gstamcaudiodec.c

```diff
@@ -449,7 +449,7 @@ gst_amc_audio_dec_open (GstAudioDecoder * decoder)
   GstAmcAudioDec *self = GST_AMC_AUDIO_DEC (decoder);
   GstAmcAudioDecClass *klass = GST_AMC_AUDIO_DEC_GET_CLASS (self);
 
-  GST_DEBUG_OBJECT (self, "Opening decoder");
```

Comment:
Is not INFO more appropiate than ERROR for the purpose?
Why "Ocucupying" would be more descriptive or clear than "Opening" to justify the change?

Reply:
When done "in a good way, as things should be done" - yes, it should be INFO level of another category that is set up to be shown in logs in fluendo-sdk.
But the problem is that we are in a big hurry, we need to deliver today. I'll try to do it if I will have time today.

So, this change is making the log appear by default in released library, so Seraphic can track when decoder is occupied by player, and when it's released. "Open" is not exact enough for this case, it's too generic.